### PR TITLE
HOME-115 adding onboarding status to routes

### PR DESCRIPTION
**Pull request by antoinemeyer**
https://trizic.atlassian.net/browse/HOME-115

isolating onboarding related things in another enum

@eweise @johntowell @LowellBateman please review

### CODE REVIEW

File: api/v2/account/schema/detail.res.json [hunk 1]
```diff
@@ -26,13 +26,19 @@
     "userStatus": {
       "type": "string",
       "enum": [
-        "EMAIL_VERIFIED",
```
Comment (antoinemeyer) [inline api/v2/account/schema/detail.res.json]: email_verified has been dead for a while I think

**Comment by johntowell** [comment]
I know it's not there now, but can we add a required section and add the required fields to it?

**Comment by antoinemeyer** [comment]
`onboardingStatus` is actually a field that can be null.

We could add `required` for the other fields but I believe that should take place in another PR.

**Comment by antoinemeyer** [comment]
That field is actually required, it's the value that can be null, I've updated the schema

**Comment by johntowell** [comment]
:+1:

**Comment by eweise** [comment]
:+1:

**Pull request merged**